### DISCUSSION TITLE
靄與霾

### DISCUSSION THREAD

**Discussion by DarsooK** [1101-GS4719-C/Python-Community-GS4719-C#3391] [General]
今日中壢下午一點的AQI指數為142「對敏感族群不健康」，快接近紅燈「有害健康」了，但即時天氣預報上的天氣狀態為「靄」而非比較常聽到的「霾」，小科普一下這兩者的差別：

靄（Mist）：能見度1公里至10公里。主要由懸浮於空氣中的微小水滴或濕性顆粒組成。和霧（Fog）相似，但空氣中水氣含量較霧低，不至於能見度降到霧的程度（霧通常能見度小於1公里）。整體環境看起來較柔和朦朧，天空呈現淡淡的灰白或帶有藍白色調。

霾（Haze）：能見度低於10公里。主要成分不是水氣，而是各類乾性細懸浮顆粒（例如PM2.5、PM10、煙塵、灰塵、工業排放等），因此空品較差。天空顏色偏灰黃或灰白，景物邊緣看起來模糊不清。

簡單來說，濕潤朦朧時為靄；乾燥混濁時為霾。而如果天空既濕潤、又混濁時，稱之為「霧霾」（Smog）。

靄與霾雖與空品有所關連，但準確程度還是要看AQI：當 AQI 較高時，核心重點在於「減少暴露、個人防護、隨時掌握資訊」，可藉由配戴適當口罩、減少戶外活動時間、維持室內空氣清淨等方式保護健康。

**Comment by christaaah** [comment]
第一次知道這些，只知道霧霾，長知識了！順帶一提冬天真的空氣品質普遍都不好，特別是南部，以前在南部的時候每年冬天只要不戴口罩在鼻子裡一定會殘留髒髒的東西，所以過敏會加重，我現在只要冬天出門我都會戴口罩，除了過濾空氣外還可以保暖，推薦！

**Comment by kzx-creat** [comment]
靄ㄞˇ，剛剛不知道怎麼唸還去google了一下，國文不好哭。我出門習慣戴口罩，天冷可以保暖，呼吸也比較舒服。
前幾個禮拜五空氣也超不好，真慶幸自己有戴口罩，要不然吸一下髒空氣，腦袋都黑白的。

**Comment by yue083139** [comment]
靄，看了上面的討論區又學了個新字真好，不過老實說最近的空氣品質感覺真的都不是太好，空氣都有點霧霧的，明明之前都沒什麼這種感覺，但最近的過敏和咳嗽變得特別嚴重，讓我有點像是回到台中的感覺，我覺得最近出門時感覺還是要稍微關注一下空氣品質決定要不要帶個口罩😷除了過濾一下髒空氣外也能夠讓臉部保持溫暖，是一個很讚的發明呢，當空氣中水分較多，能見度較高時，靄或許感覺不那麼嚴重，但高AQI數字背後的風險依然存在，所以還是要注意防護會更好。
